atomic_reactor/plugins/exit_sendmail.py

```diff
@@ -54,46 +47,65 @@ class SendMailPlugin(ExitPlugin):
     AUTO_SUCCESS = 'auto_success'
     AUTO_FAIL = 'auto_fail'
     AUTO_CANCELED = 'auto_canceled'
+    DEFAULT_SUBMITTER = 'Unknown'
 
     allowed_states = set([MANUAL_SUCCESS, MANUAL_FAIL, AUTO_SUCCESS, AUTO_FAIL, AUTO_CANCELED])
 
-    PDC_TOKEN_FILE = 'pdc.token'
-    PDC_CONTACT_ROLE = 'Devel_Owner'
-
-    def __init__(self, tasker, workflow, send_on=None, url=None, submitter='unknown', pdc_url=None,
-                 pdc_verify_cert=True, pdc_component_df_label="com.redhat.component", pdc_secret_path=None,
-                 pdc_contact_role=None, smtp_uri=None, from_address=None,
-                 error_addresses=None):
+    def __init__(self, tasker, workflow,
+                 send_on=None,
+                 url=None,
+                 smtp_uri=None,
+                 from_address=None,
+                 error_addresses=None,
+                 always_notify=None,
+                 email_domain=None,
+                 koji_hub=None,
+                 koji_proxyuser=None,
+                 koji_ssl_certs_dir=None,
+                 koji_krb_principal=None,
+                 koji_krb_keytab=None,
+                 to_koji_submitter=False,
+                 to_koji_pkgowner=False):
         """
         constructor
 
         :param tasker: DockerTasker instance
         :param workflow: DockerBuildWorkflow instance
-        :param send_on: list of build states when a notification should be sent
-        :param url: URL to OSv3 instance where the build logs are stored
-        :param submitter: name of user who submitted a build (plain string)
-        :param pdc_url: URL of PDC to query for contact information
-        :param pdc_verify_cert: whether or not to verify SSL cert of PDC (defaults to True)
-        :param pdc_component_df_label: name of Dockerfile label to use as PDC global_component
-        :param pdc_secret_path: path to pdc.token file; $SOURCE_SECRET_PATH otherwise
-        :param pdc_contact_role: name of PDC role to contact
-        :param smtp_uri: URL of SMTP server to use to send the message (e.g. "foo.com:25")
-        :param from_address: the "From" of the notification email
-        :param error_addresses: list of email addresses where to send an email if there's an error
-            (e.g. if we can't find out who to notify about the failed build)
+        :param send_on: list of str, list of build states when a notification should be sent
+        :param url: str, URL to OSv3 instance where the build logs are stored
+        :param smtp_uri: str, URL of SMTP server to use to send the message (e.g. "foo.com:25")
+        :param from_address: str, the "From" of the notification email
+        :param error_addresses: list of str, list of email addresses where to send an email if
+            there's an error (e.g. if we can't find out who to notify about the failed build)
+        :param always_notify: str, always send a message to this email address
```

Comment:
I don't mind changing it to `cc_address`, but in fact this address would be used in `To:`, so I guess it would still be confusing.

Perhaps the better way would be to use this address in CC if koji submitter/pkg owner are available and use it in `To:` if koji recipients are not set?